### PR TITLE
Feature/remove genesis sidebar layout

### LINKED ISSUE #38: Remove layout settings on all archive edit screens
We hid layouts on /wp-admin/edit.php?post_type=books&page=genesis-cpt-archive-books

However, they currently show as available, along with simple sidebars settings on other archive pages. 

Since those pages are forced to full width, those options don't work:

book author

Need to remove on each of these:
- Authors
- Series
- Tags

### PR DESCRIPTION
**Pull request by marksabbath**
This PR should fix #38 and #40.

### CODE REVIEW

**Comment by marksabbath** [comment]
Hey, @nickcernis when you have some spare time, would you mind reviewing this PR?

 I'm not really sure who else would be available for doing this, so would you mind adding another reviewer, since 2 are required to approve the PR?

File: readme.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 === Genesis Author Pro ===
-Contributors: nick_thegeek, dreamwhisper, laurenmancke, studiopress
+Contributors: nick_thegeek, dreamwhisper, laurenmancke, studiopress, marksabbath, jivedig
 Tags: genesis, authors, books
 Requires at least: 3.9
-Tested up to: 4.7
+Tested up to: 5.0.3
 Stable tag: 1.0
```
Comment (nickcernis) [inline readme.txt]: @marksabbath Should the stable tag read 1.0.1 for release (here and in readme.md)?

**Review by nickcernis — CHANGES_REQUESTED** [review]
Looks great! Nice work removing the archive meta boxes.

Just the stable tag to update, unless you want to do that in a separate PR/commit.

**Comment by nickcernis** [comment]
@dreamwhisper @mindctrl Do either of you have time to review this too?

**Review by mindctrl — APPROVED** [review]
Works as expected. Just need the readme updates mentioned by @nickcernis.

**Comment by marksabbath** [comment]
Thanks for reviewing all that guys! @nickcernis changes have been made. Would you mind taking another look?

Thanks in advance :)

**Review by nickcernis — APPROVED** [review]

**Comment by nickcernis** [comment]
Looks good, @marksabbath.

**Pull request merged**